swtril43.nv.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_notes": [
+    "Compiled by HorsePin"
+  ],
+  "_copyright": "Copyright (C) 2022 by YourName your email>",
+  "_license": "GNU Lesser General Public License v3.0",
+  "_endian": "big",
+  "_ramsize": 0,
+  "_roms": [
+    "swtril43"
+  ],
+  "_fileformat": 0.1,
+  "_version": 0.1,
+  "last_game": [
+    {"encoding":"bcd","length":4,"packed":true,"start":5351},
+    {"encoding":"bcd","length":4,"packed":true,"start":5356},
+    {"encoding":"bcd","length":4,"packed":true,"start":5361},
+    {"encoding":"bcd","length":4,"packed":true,"start":5366},
+    {"encoding":"bcd","length":4,"packed":true,"start":5371},
+    {"encoding":"bcd","length":4,"packed":true,"start":5376}
+  ],
+  "high_scores": [
+    {"label":"#1 Jedi","short_label":"#1","initials":{"start":5430,"encoding":"ch","length":3},"score":{"encoding":"bcd","length":4,"packed":true,"start":7712}},
+    {"label":"#2 Jedi","short_label":"#2","initials":{"start":5329,"encoding":"ch","length":3},"score":{"encoding":"bcd","length":4,"packed":true,"start":8036}},
+    {"label":"#3 Jedi","short_label":"#3","initials":{"start":5332,"encoding":"ch","length":3},"score":{"encoding":"bcd","length":4,"packed":true,"start":7722}},
+    {"label":"#4 Jedi","short_label":"#4","initials":{"start":5335,"encoding":"ch","length":3},"score":{"encoding":"bcd","length":4,"packed":true,"start":7727}},
+    {"label":"#5 Jedi","short_label":"#5","initials":{"start":5338,"encoding":"ch","length":3},"score":{"encoding":"bcd","length":4,"packed":true,"start":7732}},
+    {"label":"#6 Jedi","short_label":"#6","initials":{"start":5341,"encoding":"ch","length":3},"score":{"encoding":"bcd","length":4,"packed":true,"start":7737}}
```

Comment:
`packed` has always defaulted to `true`, and it's rare to even see a game where we need to use `false`.

Can you leave that out of your files?

Also, once you've updated your console app, can you remove any 0-length initials entries?

I love that you're doing this work -- please let me know if there's anything I can do to help with it.

Reply:
Should be able to, I think I would have to make a custom serializer attribute for the library. You can use JSON ignore but it's only for default values and null. Booleans default is false so it wouldn't output transformers packed, so I switched it off.

Reply:
I'm not opposed to updating the spec so all default values are `false`.  I was trying to avoid the double negative of naming the attribute `unpacked`, but it makes more sense to me to default settings to `false` instead of `true`.  I can update the file version to 0.2 with that change, and update my Python code to check either attribute.

Reply:
I think you should leave it as it is. I've done some updates to my windows tool anyway which handles all the defaults and null. Like you say so many games do default to true so I think that should be the default and not the default of the type. I even have my props set to packed = true so you don't usually have to change that.